### ISSUE TITLE
Unclear which HP devices support LVFS

### ISSUE THREAD

**Issue by mcatanzaro** [fwupd/missing-firmware-hp#3]
I'm planning to buy a laptop that is supported by LVFS, but it's currently unclear which HP laptops are actually supported. HP offers firmware for a limited subset of its devices. In most cases, the device name on LVFS corresponds to an actual understandable real-world marketing name, but in the following cases the device name is generic and difficult to make sense of:

 * [Commercial Desktop S12 Family System Update](https://fwupd.org/lvfs/devices/com.hp.desktop.S12.firmware)
 * [Commercial Desktop S21 Family System Update](https://fwupd.org/lvfs/devices/com.hp.desktop.System.S21.firmware)
 * [Commercial Laptop S70 Family System Update](https://fwupd.org/lvfs/devices/com.hp.laptop.System.S70.firmware)
 * [Commercial Laptop S73 Family System Update](https://fwupd.org/lvfs/devices/com.hp.laptop.System.S73.firmware)
 * [Commercial Laptop T91 Family System Update](https://fwupd.org/lvfs/devices/com.hp.laptop.system.T91.firmware)
 * [Commercial Laptop V90 Family System Update](https://fwupd.org/lvfs/devices/com.hp.laptop.system.V90.firmware)
 * [Desktop Workstation U60 Family System Update](https://fwupd.org/lvfs/devices/com.hp.workstation.system.U60.firmware)
 * [Desktop Workstation U61 Family System Update](https://fwupd.org/lvfs/devices/com.hp.workstation.system.U61.firmware)

It's not possible to search the web for e.g. "Commercial Desktop S12" or "Commercial Laptop V90" to figure out which HP hardware the updates correspond to. I have a theory that the V90 laptops might be part of the Victus line (but which ones?), and the S laptops might be Spectre (but which ones?), but this is just an uneducated guess. It's not clear how we could know for sure, and maybe it's a bad guess, because I don't know of any corresponding product lines that would match T01, U60, U61.

**Comment by hughsie** [comment]
Agreed. I've asked two HP vendor manager about this; if I get any update I can share I'll paste it here too. Thanks.
